src/views/Card/Card.js

```diff
@@ -55,6 +55,9 @@ export default class Card extends Component {
     /** A card can contain an Image component. */
     image: customPropTypes.itemShorthand,
 
+    /** A card can be formatted to behave like a link (hover effects, pointer cursor) */
```

Comment:
Let's go with our doc site description:
`A card can be formatted to link to other content.`

Since, the style can vary from theme to theme.